### PR TITLE
SEED-712: Languages List 

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

File: iogt/static/css/_base.scss [hunk 1]
```diff
@@ -1,31 +1,31 @@
 
 html, body {
-  margin: 0;
-  background: #fff;
+	margin: 0;
```
Comment (peterp) [inline iogt/static/css/_base.scss]: I think your editors indentation settings may be broken?

File: iogt/static/css/modules/_m-languages.scss [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  color: #fff;
+  padding-left: 2px;
+  text-decoration: none;
+  text-transform: uppercase;
+  
+  &.active {
+    color: #fff;
+  }
+
+  &:focus,
```
Comment (peterp) [inline iogt/static/css/modules/_m-languages.scss]: these can be removed.

**Comment by Mitso** [comment]
@peterp can you please review.
 @saeedmb8

File: iogt/templates/core/main.html [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
+{% load static core_tags molo_survey_tags poll_votings %}
 
 {% block logo %}
 <img src="{% static 'logo_2line.png' %}" alt="The Internet of Good Things" />
 {% endblock %}
 
 {% block content %}
     {% latest_listing_homepage %}
-    {% surveys_list %}
+    {% poll_page %}
```
Comment (smarzban) [inline iogt/templates/core/main.html]: @Mitso can you please remove `{% poll_page %}` as well

File: iogt/templates/core/main.html [hunk 4]
```diff
@@ -1,12 +1,13 @@
 {% extends "base.html" %}
 
-{% load static core_tags molo_survey_tags %}
+{% load static core_tags molo_survey_tags poll_votings %}
```
Comment (smarzban) [inline iogt/templates/core/main.html]: @Mitso also `poll_votings` from here

File: requirements.txt [hunk 5]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (smarzban) [inline requirements.txt]: @Mitso and this new line :)

File: iogt/templates/base.html [hunk 6]
```diff
@@ -21,11 +21,13 @@
         <meta property="og:image" content="{% if self.social_media_image %}{{request.site.root_url}}{{ tmp_photo.url }}{% endif %}"/>
         <meta property="og:image:url" content="{% if self.social_media_image %}{{request.site.root_url}}{{ tmp_photo.url }}{% endif %}"/>
         <meta name="viewport" content="width=device-width, initial-scale=1" />
-
         {% compress css %}
         <link rel="stylesheet" type="text/x-scss" href="{% static 'css/style.scss' %}">
         {% endcompress %}
         {% block extra_css %}{% endblock %}
+        <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.0.0-rc1/jquery.min.js"></script>
```
Comment (peterp) [inline iogt/templates/base.html]: Are we using jQuery?

File: iogt/templates/base.html [hunk 7]
```diff
@@ -21,11 +21,13 @@
...
+        <script src="https://use.fontawesome.com/57021264f5.js"></script>
```
Comment (peterp) [inline iogt/templates/base.html]: Is this for icon fonts?

File: iogt/templates/base.html [hunk 8]
```diff
@@ -41,14 +43,34 @@
       })(window,document,'script','dataLayer','{{settings.core.SiteSettings.ga_tag_manager}}');</script>
       <!-- End Google Tag Manager -->
       {% endif %}
-
+      
+      <div class="languages">
```
Comment (peterp) [inline iogt/templates/base.html]: This should be inside the `if` statement

File: iogt/templates/base.html [hunk 9]
```diff
@@ -41,14 +43,34 @@
...
-        {% endfor %}
-      </div>
+        {% get_current_language as LANGUAGE_CODE %}
+        <div class="language__current">
+          <span class="language__title language__title--current">{% trans "Site Language:" %}  </span>
+          <span class="language__title">{{LANGUAGE_CODE|language_name_local}}</span>
+        </div>
+        <label for="language_dropdown" class="language__dropdown-button">
+          <span class="language__title">Change Language</span>
+          <span class="language__title--icon">
```
Comment (peterp) [inline iogt/templates/base.html]: We should add this in with css; using an `:after` selector.

File: iogt/templates/base.html [hunk 10]
```diff
@@ -41,14 +43,34 @@
...
-          <a href="{% url 'locale_set' language.locale %}?next={{request.path}}">{{language}}</a>
-        {% endfor %}
-      </div>
+        {% get_current_language as LANGUAGE_CODE %}
+        <div class="language__current">
+          <span class="language__title language__title--current">{% trans "Site Language:" %}  </span>
+          <span class="language__title">{{LANGUAGE_CODE|language_name_local}}</span>
+        </div>
+        <label for="language_dropdown" class="language__dropdown-button">
+          <span class="language__title">Change Language</span>
```
Comment (peterp) [inline iogt/templates/base.html]: 1. The `span` tags aren't necessary if we remove the icon, and use `:after`
2. Use `{% trans "Change Language" %}`

File: iogt/templates/base.html [hunk 11]
```diff
@@ -41,14 +43,34 @@
...
+          <span class="language__title--icon">
+            <i class="fa fa-angle-down" aria-hidden="true"></i>
+          </span>
+        </label>
+
+        <ul class="language-list">
+          <input id="language_dropdown" type="checkbox" name="language-dropdown">
```
Comment (peterp) [inline iogt/templates/base.html]: We can't place an `input` inside a `ul` like this, it must be inside a `li`

File: iogt/templates/base.html [hunk 12]
```diff
@@ -41,14 +43,34 @@
...
+          <div class="language-list__toggle">
```
Comment (peterp) [inline iogt/templates/base.html]: Hmmm... We cant do this either, it isn't valid HTML. `ul` and `ol` tags can only nest `li` elements

File: iogt/templates/base.html (hunk 7 above)
Comment (Mitso) [inline iogt/templates/base.html]: Yes this is font-awesome for the dropdown. I will use an image icon once we have the assets.

File: iogt/templates/base.html [hunk 13]
```diff
@@ -21,11 +21,13 @@
...
         <meta property="og:image:url" content="{% if self.social_media_image %}{{request.site.root_url}}{{ tmp_photo.url }}{% endif %}"/>
         <meta name="viewport" content="width=device-width, initial-scale=1" />
-
         {% compress css %}
         <link rel="stylesheet" type="text/x-scss" href="{% static 'css/style.scss' %}">
         {% endcompress %}
         {% block extra_css %}{% endblock %}
+
+        
+        <script src="https://use.fontawesome.com/57021264f5.js"></script>
```
Comment (peterp) [inline iogt/templates/base.html]: I don't think we need to use images, see: http://unicode-table.com/en/sets/arrows-symbols/

**Comment by peterp** [comment]
👍

File: iogt/static/css/modules/_m-languages.scss [hunk 14]
```diff
@@ -0,0 +1,42 @@
+.languages {
+  background: $victoria;
+  color: $white;
```
Comment (peterp) [inline iogt/static/css/modules/_m-languages.scss]: `white` and `black` are already colours supported by CSS itself.

Comment (Mitso) [inline iogt/static/css/modules/_m-languages.scss]: Ok cool, I've removed the variables below.

**Comment by peterp** [comment]
👍

**Pull request merged**